src/olympia/amo/sitemap.py

```diff
@@ -454,11 +455,16 @@ def render_index_xml(sitemaps):
 
 
 def get_sitemap_path(section, app, page=1):
+    if section is None or app is None:
+        # If we don't have a section or app, we don't need a complex directory
+        # structure and we can call the first page 'sitemap' for convenience
+        # (it's likely going to be the only page).
+        endpath = str(page) if page != 1 else 'sitemap'
+    else:
+        endpath = id_to_path(page)
```

Comment:
If we're already creating subdirectories for section and app do we even need a further breakdown into subdirectories?
I.e. what's the maximum number of files a given section/app has?  (which I'm guessing is `/addons/firefox/`; I should know this already but forgot)

Reply:
I had the same thought but I looked it up and it's ~ 5000 for addons/firefox. Which is bordeline manageable, but wouldn't leave space to grow.